js/memory.js

```diff
@@ -2,8 +2,50 @@ class MemoryGame {
   constructor(cards){
     this.cards = cards;
     // add the rest of the class properties here
+    this.pickedCards = [];
+    this.pairsClicked = 0;
+    this.pairsGuessed = 0;
+  }
+
+  shuffleCards() {
```

Comment:
Is shuffleCards called somewhere? :)
The cards in the game I just played are rendered in a pattern.